README.md

```diff
@@ -19,9 +19,9 @@ system, whereas Fabric is general purpose.
 
 ## Technical Overview of Indy Plenum
 
```

Comment:
I think it makes sense to put a reference to `http://hyperledger-indy.readthedocs.io/projects/plenum` here.